src/js/tracker.js

```diff
@@ -1410,7 +1413,8 @@
 
 				// Periodic check for activity.
 				lastActivityTime = now.getTime();
-				setInterval(function heartBeat() {
+				clearInterval(pagePingInterval)
+				pagePingInterval = setInterval(function heartBeat() {
```

Comment:
It's one of only of two explanations I could come up with regarding #568, but no, no evidence.
Anyway, I don't think that could hurt the codebase since not keeping track of intervals is a pretty bad idea altogether.